### PR TITLE
Add AjaxView.

### PR DESCRIPTION
**Pull request by ADmad**
Closes cakephp/cakephp#6743

### CODE REVIEW

File: src/View/AjaxView.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    public function __construct(
+        Request $request = null,
+        Response $response = null,
+        EventManager $eventManager = null,
+        array $viewOptions = []
+    ) {
+        parent::__construct($request, $response, $eventManager, $viewOptions);
+
+        if ($response && $response instanceof Response) {
+            $response->type('ajax');
```
Comment (lorenzo) [inline src/View/AjaxView.php]: I wonder if this line would make more sense in initialize() instad of here

**Comment by ADmad** [comment]
@lorenzo Done

**Comment by markstory** [comment]
Derp I merged this without realizing that $response is undefined in initialize().

**Comment by ADmad** [comment]
Doh, stupid copy paste by me. I'll fix it.

**Comment by markstory** [comment]
@ADmad done :)

**Comment by ADmad** [comment]
Thanks :)

**Pull request merged**